app-shells/zsh/zsh-5.9.ebuild

```diff
@@ -50,6 +50,13 @@ if [[ ${PV} == *9999 ]] ; then
 		)"
 fi
 
+PATCHES=(
+	# add openrc specific options for init.d completion
+	"${FILESDIR}"/${PN}-5.3-init.d-gentoo.diff
+	# Closes: https://bugs.gentoo.org/833981
```

Comment:
small nit on the bug references "Closes: https://bugs.gentoo.org/XXX" is for the git commit message, in the ebuilds you typically just write "bug XXX"

Reply:
@laumann Thanks, I'll change.

Reply:
Should be fixed now.